# A representation of pose in free space, composed of position
# and orientation using Euler angles (typically using a right
# handed x-y-z convention i.e. rpy angles).

geometry_msgs/Point position
geometry_msgs/Vector3 orientation
